buffalo/cmd/build/assets.go

```diff
@@ -2,12 +2,14 @@ package build
 
 import (
 	"github.com/gobuffalo/buffalo/generators/assets/webpack"
+	"github.com/gobuffalo/envy"
 	pack "github.com/gobuffalo/packr/builder"
 	"github.com/pkg/errors"
 )
 
 func (b *Builder) buildAssets() error {
 	if b.WithWebpack {
+		envy.Set("NODE_ENV", "production")
```

Comment:
hi Markbates, 
would be nice if it detects GO_ENV and set it  with default to "production"
and set it into NODE_ENV